db/models/user.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const { Model } = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class User extends Model {
+    static associate({ Event, UserEvent }) {
```

Comment:
it's better to have all associations in separate place. It can be function, for example. This makes it easier to handle changes for them in future.